aztec/src/test/kotlin/org/wordpress/aztec/AztecToolbarTest.kt

```diff
@@ -557,7 +603,7 @@ class AztecToolbarTest {
         Assert.assertFalse(italicButton.isChecked)
 
         italicButton.performClick()
-        Assert.assertEquals("<b>bol</b><b><i>ditalic</i></b>", editText.toHtml())
+        Assert.assertEquals("<b>bol</b><b><em>ditalic</em></b>", editText.toHtml())
```

Comment:
I'm not sure why we're changing this test? All the other tests for italic text in this file use `<i>`, so we should stay consistent here.

Reply:
This was changed by the 1st person to work on this PR (it contains code from myself plus 2 others). I'm not sure what the intention was originally, but digging into this helped me uncover some bugs that I have now fixed in my 2 most recent commits. After fixing them, I was able to revert this.